feature-configs/demo/sctp/sctp_module_mc_patch.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: sctp-worker
```

Comment:
what should this label actually be if there's no sctp-worker?

Reply:
that's the machine config role, what the admin would put in the list of configs selected by the unique machine config pool. It's not related in any way to the node.
The binding with the node is done by the machine config pool via the selector.